### DISCUSSION TITLE
[Feature] Home Assistant Integration to slideshow on Smartscreens

### DISCUSSION THREAD

**Discussion by fabioteixei** [immich-app/immich#6199] [Feature Request]
### The feature

The goal is to use Immich to replace for good Google Photos or Amazon Photos, but to do that we need an way to do an slide show of favorite pictures or even an albun and display that slide show on a Smartscreen like Echo Show or Google Home Hub using home assistant as the broker (since Echo Show or Google Home Hub will never natively support Immich).

I dont think you need to wory abou the home assistant side because home assistant already have methods to display media on those screens. I think what you need is to focus in some kind of integration to pass to home assistant an HTTP playlist, i dont know, maybe an m3u playlist or some way to dinamicaly transform an slideshow to an mp4, something like that.

An more complete integration would be great, but a simple one can do the trick to help increase the Wife aproval factor of ditching Google Photos to use Immich.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Related: #1614

**Comment by mulder82** [comment]
Check this: https://github.com/mulder82/immich-slideshow

**Comment by chris-burrow-apps** [comment]
There is also this add on: https://github.com/outadoc/immich-home-assistant

**Comment by chris-burrow-apps** [comment]
This is a cool app I discovered: https://github.com/damongolding/immich-kiosk

**Reply by PKort** [reply]
I second that.
I ended up using: 
1) Immich Kiosk to access Immich photos (selected albums) and display them as a webpage. I added password protection so accessing root site returns unauthorized error.
2) nginx reverse proxy to add SSL to Immich Kiosk website
3) On HA side I decided to go with grid layout where most of the display is used by IFRAME card pointing to Immich Kiosk SSL website with password added as parameter. Small parts of the grid are used by buttons. Each button displays a bubble card popup with respective controls (sensors/buttons) to allow control of the home.

**Comment by bo0tzz** [comment]
Closing this as there are great community projects that cover the need.
